jake/ossindex/ossindex.py

```diff
@@ -47,18 +64,70 @@ def chunk(self, purls):
             chunks.append(divided)
         return chunks
 
-    def callOSSIndex(self, purls):
-        self._log.debug(purls)
+    def callOSSIndex(self, purls: Coordinates):
+        self._log.debug("Purls received, total purls before chunk: %s", len(purls.get_coordinates()))
```

Comment:
I like the debug info!

Reply:
When running a lot of the debug stuff was just showing what class it was, etc..., not super helpful, this was my attempt at making it much more useful for someone.